packages/test/snapshots/src/replayMultipleFiles.ts

```diff
@@ -242,6 +262,31 @@ async function processNodeForWrite(
     fs.writeFileSync(versionFileName, JSON.stringify({ snapshotVersion: pkgVersion }), { encoding: "utf-8" });
 }
 
+/**
+ * In NewSnapshots mode, new test snapshot is being added. We need to run the replay tool in "write" mode which will
+ * generate snapshot files and write them to the current snapshots dir.
+ */
+async function processNodeForNewSnapshots(
+    data: IWorkerArgs,
+    concurrently: boolean,
+    limiter: ConcurrencyLimiter,
+) {
+    const currentSnapshotsDir = `${data.folder}/${currentSnapshots}`;
+    // If current snapshots dir already exists, there is nothing to do.
+    if (fs.existsSync(currentSnapshotsDir)) {
```

Comment:
Just to make sure I'm following, we assume that the folder existing means that it is also populated with the generated data?

Reply:
Yes, exactly. If we let it run, it will update the current_snapshots for existing snapshots too. I will add a comment to make it clear.